package.json

```diff
@@ -48,11 +48,12 @@
   ],
   "dependencies": {
     "react-resize-detector": "^7.1.2",
+    "semantic-ui-css": "^2.5.0",
+    "semantic-ui-react": "^2.1.4",
```

Comment:
my hope is we avoid adding any dependencies. I spent a lot of time removing dependencies to keep this set as small as possible to reduce the size of the bundle + minimize time spent on dependency management because of incompatibilities. Eg I can see `semantic-ui-react ^2` suddenly not support the newest version of React when we bump React, then we've got to bump semantic-ui-react, then we've got to update our PrimerDesign components to update React. I'd love to get rid of `react-resize-detector` and `webfontloader` as well but wasn't able to last I checked